packages/flutter_tools/lib/src/cache.dart

```diff
@@ -50,6 +52,16 @@ class Cache {
     stampFile.writeAsStringSync(version);
   }
 
+  Future<Null> updateAll() async {
+    MaterialFonts materialFonts = new MaterialFonts(cache);
+    if (!materialFonts.isUpToDate())
+      await materialFonts.download();
+
+    FlutterEngine engine = new FlutterEngine(cache);
+    if (!engine.isUpToDate())
+      await engine.download();
+  }
```

Comment:
I'd prefer not to :grimacing: ; which might sound odd, but the progress display will work better if there's just one task on-going at a time, and the total download time w/ an engine rev is just ~10 seconds.